test/e2e/template/busyboxds.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: extensions/v1beta1
+kind: DaemonSet
+metadata:
+  name: busybox
+  namespace: default
+spec:
+  template:
+    metadata:
+      labels:
+        component: busybox
+    spec:
+      hostNetwork: true
+      containers:
+      - name: busybox
+        image: alpine:3.8
```

Comment:
just wanted to confirm that its the newest version ?

Reply:
The latest version is `alpine:3.9.4`. I kept it at `3.8` because that is the version we are using to build other images.